arangod/Pregel/Status/ConductorStatus.h

```diff
@@ -33,34 +34,45 @@
 #include <Cluster/ClusterTypes.h>
 
 #include <Pregel/Common.h>
-#include <Pregel/Status/WorkerStatus.h>
+#include <Pregel/Status/Status.h>
 
 namespace arangodb::pregel {
 
-struct ConductorStatus {
-  TimeStamp timeStamp = std::chrono::system_clock::now();
-  std::unordered_map<ServerID, WorkerStatus> workers;
+struct AccumulatedConductorStatus {
+  Status status;
+  std::unordered_map<ServerID, Status> workers;
+  bool operator==(AccumulatedConductorStatus const&) const = default;
+};
+
+template<typename Inspector>
+auto inspect(Inspector& f, AccumulatedConductorStatus& x) {
+  return f.object(x).fields(
+      f.field("aggregatedStatus",
+              x.status),  // TODO status entries should be on top level
+      f.field("workerStatus", x.workers));
+}
 
+struct ConductorStatus {
+  std::unordered_map<ServerID, Status> workers;
```

Comment:
Are `workers` used for anything except iterating over them? If no, we could take `vector` instead of `unordered_set`.

Reply:
The map is the correct solution; the only thing you *could* do is to use `FlatHashMap`